src/styles/rating-stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background: no-repeat center url(/src/images/star.svg);
```

Comment:
Consider using a variable for the URL path to the star image if it is used in multiple places. This can improve maintainability and make it easier to update the image path in the future.